build.xml

```diff
@@ -444,7 +453,7 @@
 
     <target name="coverage" depends="send-java-coverage,send-somns-coverage" />
 
-    <target name="tests" depends="core-tests,replay-tests,coverage">
+    <target name="tests" depends="core-tests,serialization-tests,replay-tests,coverage">
```

Comment:
This target is not tested on Travis or GitLab CI. Depending on how long the tests takes, I'd add them to one of the existing test jobs.
Possibly to one of the cases here: https://github.com/smarr/SOMns/blob/dev/.travis.yml#L67
`ant serialization-tests` or the call to `./som ...` directly.